main.go

```diff
@@ -150,32 +100,32 @@ func main() {
 	router.Handle("/redir/{data:.*}", searchHandler)
 	router.Handle("/download/{uri:.*}", createReverseProxy("download", downloaderURL))
 
-	if config.DatasetRoutesEnabled == true {
+	if cfg.DatasetRoutesEnabled == true {
 		router.Handle("/datasets/{uri:.*}", createReverseProxy("datasets", datasetControllerURL))
 		router.Handle("/feedback{uri:.*}", createReverseProxy("feedback", datasetControllerURL))
 		router.Handle("/filters/{uri:.*}", createReverseProxy("filters", filterDatasetControllerURL))
 		router.Handle("/filter-outputs/{uri:.*}", createReverseProxy("filter-output", filterDatasetControllerURL))
 	}
 	// remove geo from prod
-	if config.GeographyEnabled == true {
+	if cfg.GeographyEnabled == true {
 		router.Handle("/geography{uri:.*}", createReverseProxy("geography", geographyControllerURL))
 	}
 	router.Handle("/{uri:.*}", reverseProxy)
 
 	log.Event(nil, "Starting server", log.Data{
-		"bind_addr":                config.BindAddr,
-		"babbage_url":              config.BabbageURL,
-		"dataset_controller_url":   config.DatasetControllerURL,
-		"geography_controller_url": config.GeographyControllerURL,
-		"downloader_url":           config.DownloaderURL,
-		"renderer_url":             config.RendererURL,
-		"site_domain":              config.SiteDomain,
-		"assets_path":              config.PatternLibraryAssetsPath,
-		"analytics_sqs_url":        config.SQSAnalyticsURL,
+		"bind_addr":                cfg.BindAddr,
```

Comment:
I think you can just pass the `cfg` struct into log.Data and it will handle the JSON marshal for you.